plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -230,24 +230,29 @@ class NetworkingFormSection extends mixin(StoreMixin) {
 
     let address = vip;
 
-    if (address == null) {
-      let port = "";
-      if (!portsAutoAssign && !ValidatorUtil.isEmpty(hostPort)) {
-        port = hostPort;
-      }
-      if (!ValidatorUtil.isEmpty(containerPort)) {
-        port = containerPort;
-      }
-      if (!ValidatorUtil.isEmpty(vipPort)) {
-        port = vipPort;
-      }
+    let port = "";
+    if (!portsAutoAssign && !ValidatorUtil.isEmpty(hostPort)) {
+      port = hostPort;
+    }
+    if (!ValidatorUtil.isEmpty(containerPort)) {
+      port = containerPort;
+    }
+    if (!ValidatorUtil.isEmpty(vipPort)) {
+      port = vipPort;
+    }
 
+    if (address == null) {
       address = `${id}:${port}`;
     }
 
+    const vipMatch = address.match(/(.+):\d+/);
+    if (vipMatch) {
+      address = `${vipMatch[1]}:${port}`;
```

Comment:
The integration test failure was on this line, because I said :${vipPort} instead of $:{port}.

It was a bad copy and paste from the other place.